.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
This has been resolved by adding `"prepare": "husky install"` to scripts section. This will initialize husky when you install it.
Refer [this documentation](https://typicode.github.io/husky/#/)